src/shared/components/results/ProteomeSuggestion.tsx

```diff
@@ -6,41 +7,84 @@ import useDataApi from '../../hooks/useDataApi';
 import { Namespace } from '../../types/namespaces';
 import { SearchResults } from '../../types/results';
 import { stringifyUrl } from '../../utils/url';
+import EntryTypeIcon from '../entry/EntryTypeIcon';
 import { SearchTextLink } from './SearchTextLink';
 
 const ProteomeSuggestion = ({
-  query,
   organismID,
+  namespace,
 }: {
-  query: string;
   organismID: string;
+  namespace: Namespace;
 }) => {
-  const [proteomeInfo, setProteomeInfo] = useState<ProteomesAPIModel>();
+  const [proteomeInfo, setProteomeInfo] = useState<
+    ProteomesAPIModel | ProteomesAPIModel[]
+  >();
 
   const { data } = useDataApi<SearchResults<ProteomesAPIModel>>(
     stringifyUrl(apiUrls.search.searchPrefix(Namespace.proteomes), {
       query: `organism_id:${organismID}`,
-      fields: 'upid',
+      fields: 'upid,organism',
+      size: 500, // to get all proteomes for UniParc
     })
   );
 
   useEffect(() => {
     if (data?.results.length) {
-      setProteomeInfo(data.results[0]);
+      if (namespace === Namespace.uniprotkb) {
+        setProteomeInfo(data.results[0]);
+      }
+      if (namespace === Namespace.uniparc) {
```

Comment:
```suggestion
      else if (namespace === Namespace.uniparc) {
```
Just to avoid the unnecessary check.